packages/babel-core/src/config/files/configuration.js

```diff
@@ -198,8 +198,9 @@ const readConfigJS = makeStrongCache(
 
 const packageToBabelConfig = makeWeakCache(
   (file: ConfigFile): ConfigFile | null => {
-    if (typeof file.options.babel === "undefined") return null;
-    const babel = file.options.babel;
+    const babel = file.options[("babel": string)];
```

Comment:
Does `const babel: string | void = file.options.babel` work? (If it doesn't it's ok - it's just that this seems too hacky to me)

Reply:
Nope :(

```sh
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ packages/babel-core/src/config/files/configuration.js:201:47

Cannot get file.options.babel because property babel is missing in object type [1].

     packages/babel-core/src/config/files/configuration.js
     198│
     199│ const packageToBabelConfig = makeWeakCache(
     200│   (file: ConfigFile): ConfigFile | null => {
     201│     const babel: string | void = file.options.babel;
```

Reply:
Oh nevermind